python/tk_multi_launchapp/base_launcher.py

```diff
@@ -116,11 +116,18 @@ def _register_launch_command(
                 "engine_name": app_engine,
             }
 
-            properties["software_entity_id"] = software_entity_id
+            properties["software_entity"] = software_entity
```

Comment:
```suggestion
            properties["software_entity_id"] = software_entity.get("id")
```
This needs to stay in, the browser integration will use that information when caching actions!